src/insert/spacefinder/article.ts

```diff
@@ -212,6 +219,10 @@ const addDesktopInline1 = (fillSlot: FillAdSlot): Promise<boolean> => {
 const addDesktopRightRailAds = (fillSlot: FillAdSlot): Promise<boolean> => {
 	let minAbove = 1000;
 
+	if (isInDeeplyReadMostViewedVariant) {
```

Comment:
I think this will lead to an extra large `minAbove` when `isPaidContent` is `true`

Reply:
Thanks Jake, good spot. I've updated the code to not add the extra space twice when the Most Viewed island is missing